packit-ci.fmf

```diff
@@ -13,6 +13,10 @@ discover:
      - /setup/install_upstream_keylime
      - /functional/basic-attestation-on-localhost
      - /functional/keylime_tenant-commands-on-localhost
+     - /functional/tpm_policy-sanity-on-localhost
```

Comment:
Are these packit tests being added as part of this commit? Or was that just for testing to see if this change fixes those tests?